packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -320,31 +335,94 @@ function WorklistDataGrid(props) {
   );
 
   const handleClearRowSelection = () => {
+    let message = '';
+    rowSelection.current.rangeUsingLastIndividuallySelectedRow = null;
     if (isAnyRowSelected()) {
-      setAriaLiveMessage(intl.formatMessage({ id: 'Terra.worklist-data-grid.all-rows-unselected' }));
+      message = intl.formatMessage({ id: 'Terra.worklist-data-grid.all-rows-unselected' });
       // Esc (while in row selection mode and rows are selected): Clear selection
       if (onClearSelectedRows) {
         onClearSelectedRows();
       }
     } else if (onDisableSelectableRows) {
-      setAriaLiveMessage(intl.formatMessage({ id: 'Terra.worklist-data-grid.row-selection-mode-disabled' }));
+      rowSelection.current.rangeUsingFocusedRow = null;
+      message = intl.formatMessage({ id: 'Terra.worklist-data-grid.row-selection-mode-disabled' });
```

Comment:
Is row selection mode being disabled or turned off?  Is this text from our requirements?

Reply:
The requirements do not specify any messages and we did not hear any objections to the current messages during our demos or UX reviews.

Reply:
But the whole strategy is supposed to be updating such that we are providing message based on input from consumers.  Should we be adding more of these messages?  What does it mean for the row selection mode to be disabled?